execution/gethexec/api.go

```diff
@@ -332,16 +334,46 @@ func (api *ArbTraceForwarderAPI) forward(ctx context.Context, method string, arg
 	return resp, nil
 }
 
-func (api *ArbTraceForwarderAPI) Call(ctx context.Context, callArgs json.RawMessage, traceTypes json.RawMessage, blockNum json.RawMessage) (*json.RawMessage, error) {
-	return api.forward(ctx, "arbtrace_call", callArgs, traceTypes, blockNum)
+func (api *ArbTraceForwarderAPI) ClipToPostNitroGenesis(blockNumOrHash json.RawMessage) (json.RawMessage, error) {
```

Comment:
actually, here we need to do the opposite.
arbtrace requests are only supported by classic node and will be forwarded to it, so they are only o.k if they are before nitro genesis.
We can reject call for any number that's below 0 (latest / safe / confirmed) with an error saying we don't support that block number for this call, and we can reject any block above chainconfig.ArbitrumChainParams.GenesisBlockNum

Reply:
Done.